test/DataStoreTests.jl

```diff
@@ -1,52 +1,85 @@
-using DistributedFactorGraphs
+if false
```

Comment:
Is the plan to ignore tests until the next PR?

Reply:
Yes, I ran into problems with tests. I wanted to get the name change in ASAP as it will influence you.